jina/__init__.py

```diff
@@ -203,6 +203,11 @@ def _set_nofile(nofile_atleast=4096):
 # Executor
 from jina.serve.executors import BaseExecutor as Executor
 from jina.serve.executors.decorators import monitor, requests
+from jina.serve.runtimes.gateway.grpc.gateway import GRPCGateway
```

Comment:
we do not want to expose this in `__init__` at all.

Reply:
We may want to expose `BaseGateway` as `Gateway` similar to Executor

Reply:
done